lib/jxl/transfer_functions-inl.h

```diff
@@ -165,7 +165,7 @@ class TF_709 {
 // Perceptual Quantization
 class TF_PQ {
  public:
-  explicit TF_PQ(float display_intensity_target)
+  explicit TF_PQ(float display_intensity_target = 255.0)
```

Comment:
... it didn't work

Reply:
Is it because of the `explicit`? Because then we could have the explicit one without a default value for the argument, and then add a no-parameter constructor like this:

```
TF_PQ() : TF_PQ(kDefaultIntensityTarget) {}
```